src/Rolodex.tsx

```diff
@@ -255,9 +255,14 @@ export default function Rolodex() {
                 const q = query.toLocaleLowerCase();
                 return (
                   c.callsign.toLocaleLowerCase().includes(q) ||
-                  c.name.toLocaleLowerCase().includes(q)
+                  c.name.toLocaleLowerCase().includes(q) ||
+                  // this may cause issues with older cards that may have an undefined location...
```

Comment:
we really don't want to break compatibility here, there are a handful of folks who use this already and breaking the site for them isn't great

Reply:
I'd argue that finding a way to not break compatibility while also being able to keep this would be preferred